sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/ConversationAnalysisClientLiveTests.cs

```diff
@@ -40,13 +40,13 @@ public async Task AnalyzeConversation()
                 kind = "Conversation",
             };
 
-            Response response = await Client.AnalyzeConversationAsync(RequestContent.Create(data));
+            Response response = await Client.AnalyzeConversationAsync(RequestContent.Create(data, new RawContentOptions() { UseCamelCaseNamingConvention = true }));
```

Comment:
this seems verbose.

Reply:
Cool, I'll go back to the single-enum value.

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-net/pull/36887